doc/getting_started/getting_started.rst

```diff
@@ -70,27 +70,19 @@ Use the following procedures to create a new development environment.
 Building and Running an Application
 ***********************************
 
-Using the 'Hello World' sample application as a base model, the following
```

Comment:
I'm not fond of having a heading with no content after them except for another heading.  Can we put this first paragraph back in?

Reply:
Hm, I don't think we ought to put it back in as-is. As I tried to explain in the commit log, this paragraph contains factual errors. I'll try to put something better in here.